newdle/client/src/components/ErrorMessage.js

```diff
@@ -2,6 +2,7 @@ import React, {useEffect} from 'react';
 import {useDispatch} from 'react-redux';
 import PropTypes from 'prop-types';
 import {useHistory} from 'react-router';
+import {Trans, Plural, t} from '@lingui/macro';
```

Comment:
Please remove the unused imports. You are only using `t` in here.

```suggestion
import {t} from '@lingui/macro';
```

Reply:
ok